day-trader-java-app/daytrader-gateway/src/main/java/com/ofss/daytrader/gateway/controller/GatewayController.java

```diff
@@ -737,7 +761,152 @@
+		String userKey = "holdings_" + userId;
+
+		if (CachedObjectsClass.getInstance().checkCacheForObject(userKey) != null) {
+			System.out.println("data is displayed from cache");
+			HoldingDataBean holdingDataBean = (HoldingDataBean) CachedObjectsClass.getInstance()
+					.checkCacheForObject(userKey);
+			return new ResponseEntity<Collection<HoldingDataBean>>((Collection<HoldingDataBean>) holdingDataBean,
+					getNoCacheHeaders(), HttpStatus.OK);
+
+		} else {
+
+			holdings.add(new HoldingDataBean(701, 107d, new BigDecimal(10), new Timestamp(System.currentTimeMillis()),
+					"s:701"));
+			holdings.add(new HoldingDataBean(702, 17d, new BigDecimal(10), new Timestamp(System.currentTimeMillis()),
+					"s:702"));
+			holdings.add(new HoldingDataBean(703, 177d, new BigDecimal(10), new Timestamp(System.currentTimeMillis()),
+					"s:703"));
+			return new ResponseEntity<Collection<HoldingDataBean>>(holdings, getNoCacheHeaders(), HttpStatus.OK);
+		}
+	}
+
+	public ResponseEntity<QuoteDataBean> getQuoteFallback(@PathVariable("symbol") String symbol) {
+		String quotekey = "getQuote_" + symbol;
+
+		if (CachedObjectsClass.getInstance().checkCacheForObject(quotekey) != null) {
+			System.out.println("data is displayed from cache");
+			QuoteDataBean quoteDataBean = (QuoteDataBean) CachedObjectsClass.getInstance()
+					.checkCacheForObject(quotekey);
+			return new ResponseEntity<QuoteDataBean>(quoteDataBean, getNoCacheHeaders(), HttpStatus.OK);
+
+		} else {
+			QuoteDataBean quoteData = new QuoteDataBean("s:701", "Company701", 9.00, new BigDecimal(195),
+					new BigDecimal(195), new BigDecimal(1), new BigDecimal(10), 1.0);
+
+			return new ResponseEntity<QuoteDataBean>(quoteData, getNoCacheHeaders(), HttpStatus.OK);
+		}
+	}
+
+	public ResponseEntity<Collection<OrderDataBean>> getOrdersFallback(@PathVariable("userId") String userId) {
+		Collection<OrderDataBean> orders = null;
+		if (CachedObjectsClass.getInstance().checkCacheForObject(userId) != null) {
+			System.out.println("data is displayed from cache");
+			QuoteDataBean quoteDataBean = (QuoteDataBean) CachedObjectsClass.getInstance().checkCacheForObject(userId);
+			return new ResponseEntity<Collection<OrderDataBean>>(orders, getNoCacheHeaders(), HttpStatus.OK);
+
+		} else {
+
+			return new ResponseEntity<Collection<OrderDataBean>>(orders, getNoCacheHeaders(), HttpStatus.OK);
+		}
+
+	}
+
+	public ResponseEntity<AccountDataBean> getAccountDataFallback(@PathVariable("userId") String userId) {
+		String DataKey = "AccountData" + userId;
+		AccountDataBean accountData = new AccountDataBean();
+		if (CachedObjectsClass.getInstance().checkCacheForObject(DataKey) != null) {
+			System.out.println("data is displayed from cache");
+			accountData = (AccountDataBean) CachedObjectsClass.getInstance().checkCacheForObject(DataKey);
+			return new ResponseEntity<AccountDataBean>(accountData, getNoCacheHeaders(), HttpStatus.OK);
```

Comment:
this line and another same line in this method, move it to the end of the method, so that the line appears only once

Reply:
changes are done as mentioned